### ISSUE TITLE
01. WAP To Print First 10 Odd Number's ? 

### ISSUE THREAD

**Issue by MadJokkerr** [MadJokkerr/Hacktoberfest-2022#2]

**Comment by ashutoshmp1** [comment]
Hey, can I contribute here and on other issues?

**Comment by arkapg211002** [comment]
I want to contribute in this in java can u pls assign me

**Comment by MadJokkerr** [comment]
@arkapg211002 go a head

**Comment by Deepanshu0509** [comment]
I would like to contribute in C++ and python language. Can you assign me this issue?

**Comment by MadJokkerr** [comment]
@arkapg211002 please make another pull request without adding name in contributors you don't have to add your name every time in contributors

**Comment by MadJokkerr** [comment]
> I would like to contribute in C++ and python language. Can you assign me this issue?

yes go a head @Deepanshu0509 contribute in c++ also follow contributions and readme guides

**Comment by arkapg211002** [comment]
> @arkapg211002 please make another pull request without adding name in contributors you don't have to add your name every time in contributors

@MadJokkerr Hello I have made another pr as stated [Issue 2](https://github.com/MadJokkerr/Hacktoberfest-2022/pull/70)

**Comment by MadJokkerr** [comment]
> > @arkapg211002 please make another pull request without adding name in contributors you don't have to add your name every time in contributors
> 
> @MadJokkerr Hello I have made another pr as stated [Issue 2](https://github.com/MadJokkerr/Hacktoberfest-2022/pull/70)

if you make a pull request with some wrong values and i reject it and if you add another program after that the next pull request will take the new program with it as well as the old program with makes clash between versions i request to fork the repo again and work properly

**Comment by arkapg211002** [comment]
Ok thanks for the information

**Comment by harkar20** [comment]
hey! i want to work on this issue, please assign it to me

**Comment by adityaonstage** [comment]
heya. i want work on this issue. i can contribute in python java as well as c++.

**Comment by MadJokkerr** [comment]
> hey! i want to work on this issue, please assign it to me

which lang in which you are comfortable in

**Comment by MadJokkerr** [comment]
> heya. i want work on this issue. i can contribute in python java as well as c++.

go a head with c++

**Comment by Faizan96322** [comment]
hey, I would like to work on this issue in C language, pls assign me this task!

**Comment by MadJokkerr** [comment]
> heya. i want work on this issue. i can contribute in python java as well as c++.

ohk go a head with c++ @adityaonstage

**Comment by MadJokkerr** [comment]
> hey, I would like to work on this issue in C language, pls assign me this task!

sorry we don't have c language

**Comment by MadJokkerr** [comment]
> heya. i want work on this issue. i can contribute in python java as well as c++.

@adityaonstage go with python lang ohk
